run_task.py

```diff
@@ -3,46 +3,84 @@
 import time
 from typing import List
 
+import click
 import pandas as pd
 import vantage6.client as vtgclient
 
-USERNAME = 'admin'
-PASSWORD = 'admin'
-
-POST = 'POST'
-WAIT_TIME = 1
-RETRIES = 20
-
-HOST = 'http://localhost'
-PORT = 5001
-
-IMAGE = 'localhost:5000/v6-carrier-py'
-METHOD = 'correlation_matrix'
-COLLABORATION_ID = 1
-ORGANIZATION_IDS = [1]  # [2, 3, 6]
-MASTER = True
-NUM_NODES = 1
-
-
-def main():
-    client = vtgclient.Client(HOST, PORT)
-    client.authenticate(USERNAME, PASSWORD)
+DEFAULT_USERNAME = 'admin'
+DEFAULT_PASSWORD = 'admin'
+
+DEFAULT_WAIT_TIME = 1
+DEFAULT_NUM_RETRIES = 20
+
+DEFAULT_HOST = 'http://localhost'
+DEFAULT_PORT = 5001
+
+DEFAULT_IMAGE = 'localhost:5000/v6-carrier-py'
+DEFAULT_METHOD = 'correlation_matrix'
+DEFAULT_COLLABORATION_ID = '1'
+DEFAULT_ORGANIZATION_IDS = '1'
+
+
+@click.command(context_settings={'ignore_unknown_options': True,
+                                 'allow_extra_args': True},
+               help='Run a task on vantage6 nodes. Optionally pass extra'
+                    'keyword arguments to be passed to the algorithm'
+                    '(i.e. "--key value")')
+@click.option('--method', default=DEFAULT_METHOD,
```

Comment:
I'd say there doesn't really have to be a default method or image, it's up to the user